src/compareNumber.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+function compareNumbers(userNumber, randomNumber) {
+  const bullsAndCows = {
+    bull: 0,
+    cow: 0,
+  };
+
+  for (let i = 0; i < userNumber.length; i++) {
```

Comment:
Try to avoid using old traditional `for` loops. Instead, use specific ones like filter/map/foreach/reduce. Here fits `reduce`.